core/domain/classifier_services_test.py

```diff
@@ -19,18 +19,20 @@
 from core.domain import classifier_registry
 from core.domain import classifier_services
 from core.domain import exp_services
+from core.platform import models
 from core.tests import test_utils
 import feconf
 import utils
 
+(classifier_models,) = models.Registry.import_models([models.NAMES.classifier])
+
 class ClassifierServicesTests(test_utils.GenericTestBase):
```

Comment:
There is a  linting issue that I checked in travis - "ensure that this file ends with exactly one newline char".